x-pack/plugins/session_view/common/mock/app_context_render.tsx

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
Personally I don't view this file and `app_root_provider.tsx` as mocks, since they're test fixtures/scaffolding

Also since these are frontend tests, maybe we should move it to a `public/test_utils.tsx` file or something? 

I envision mocks to be mock data and things like that.

Reply:
it all that makes sense to me